packages/valory/connections/ledger/connection.py

```diff
@@ -67,9 +63,9 @@ def __init__(self, **kwargs: Any):
         )
 
     @property
-    def event_new_receiving_task(self) -> asyncio.Event:
-        """Get the event to notify the 'receive' method of new receiving tasks."""
-        return cast(asyncio.Event, self._event_new_receiving_task)
+    def response_envelopes(self) -> asyncio.Queue:
+        """Get the response envelopes. Only intended to be accessed when connected."""
+        return cast(asyncio.Queue, self._response_envelopes)
```

Comment:
```suggestion
        if self._response_envelopes is None:
            raise ValueError("asyncio.Queue not set")
        return self._response_envelopes
```

Reply:
Done in d2ac6547dd2d77e3e009ce936ecbe8d4a027f5a5.